### PR TITLE
Venu/arc ds msi

### PR DESCRIPTION
**Pull request by v-vgujjula**
MSI and Memory parameter integrated on existing changes.

### CODE REVIEW

File: testsuite/azure-arc-conformance.properties [hunk 1]
```diff
@@ -37,7 +37,8 @@ azure-arc-platform.CUSTOM_LOCATION_OID=8e103bf2-1f6a-43d9-9473-52a2461574b4
 #azure-arc-agent-cleanup.PROXY_CERT_NAMESPACE="namespace of sonobuoy secret"
 
 # azure-arc-ds-connect-platform plugin configurations
-# Set below to true if you want to run the tests for data services in direct mode.
+# To perform data services with direc connect mode please enable azure-arc-platform.enable=true
```
Comment (anraghun) [inline testsuite/azure-arc-conformance.properties:40]: ~direc~ direct

File: testsuite/azure-arc-conformance.properties [hunk 2]
```diff
@@ -37,7 +37,8 @@ azure-arc-platform.CUSTOM_LOCATION_OID=8e103bf2-1f6a-43d9-9473-52a2461574b4
...
+# Set below to true if you want to run the tests for data services in direct connect mode.
```
Comment (anraghun) [inline testsuite/azure-arc-conformance.properties:41]: Line 40 and 41 are saying the same thing.

File: testsuite/conformance-test-plugins/azure-arc-agent-cleanup.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ sonobuoy-config:
   plugin-name: azure-arc-agent-cleanup
```
Comment (anraghun) [inline testsuite/conformance-test-plugins/azure-arc-agent-cleanup.yaml:3]: This change is already present in the target branch. You can remove this change

File: testsuite/conformance-test-plugins/azure-arc-platform.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 podSpec:
```
Comment (anraghun) [inline testsuite/conformance-test-plugins/azure-arc-platform.yaml:1]: This change is already present in the target branch. You can remove this change

**Review by anraghun — CHANGES_REQUESTED** [review]

File: testsuite/azure-arc-conformance.properties (hunk 1 above)
Comment (v-vgujjula) [reply testsuite/azure-arc-conformance.properties]: taken care

File: testsuite/azure-arc-conformance.properties (hunk 2 above)
Comment (v-vgujjula) [reply testsuite/azure-arc-conformance.properties]: Line 40 is prerequisite for data services with connect mode plugin.
Line 41 is just informing to enable azure-arc-ds-connect-platform plugin for data services with connect mode plugin.

File: testsuite/conformance-test-plugins/azure-arc-agent-cleanup.yaml (hunk 3 above)
Comment (v-vgujjula) [reply testsuite/conformance-test-plugins/azure-arc-agent-cleanup.yaml]: taken care

File: testsuite/conformance-test-plugins/azure-arc-platform.yaml (hunk 4 above)
Comment (v-vgujjula) [reply testsuite/conformance-test-plugins/azure-arc-platform.yaml:1]: taken care

**Pull request merged**